contents/blog/rome-hackathon.md

```diff
@@ -0,0 +1,80 @@
+
+To limit this happening, Paul worked on “FaceHog,” a notification log of changes to insights, dashboards, feature flags, and experiments you have created.
+
+![Facehog](../images/blog/rome-hackathon/facehog.gif)
+
+Now if your evil twin decides to change or delete your insights or roll out your feature to more people, you’ll get a notification.
+
+You can find more of the details in the [PR here](https://github.com/PostHog/posthog/pull/12037). 
+
+## Person feed
+
+The person page currently shows properties, events, session recordings, and more all in separate tabs. Michael and Cory felt it would be better to see all of that data at a glance. They designed and built the person feed to help achieve this.
+
+![Person feed](../images/blog/rome-hackathon/persons-feed.png)
+
+The person feed is a chronological feed of a person’s behavior. It amalgamates sessions, recordings, events, and properties onto one page. PostHog users can see the events that constitute a session along with a session recording if it exists. They can see the properties of the person in the sidebar as they go, giving them context on that person's events.
+
+Potential future improvements include pinning the most relevant property fields. This feature is still a work in progress, but you can see the [draft PR here](https://github.com/PostHog/posthog/pull/12053).
+
+## Hedgehog mode
+
+To increase the amount of fun in PostHog, Ben and Lottie worked on Hedgehog mode. Hedgehog mode spawns a hedgehog to hang out and walk around on your page. Internal testing shows it helps meet important team KPIs like increased joy, lower stress, and more hedgehog thoughts.
+
+![Hedgehog mode](../images/blog/rome-hackathon/hedgehog-mode.gif)
+
+Some day the hedgehog might help you identify popular areas and potential areas for creating actions, but for now they are just hanging out. See the [PR here](https://github.com/PostHog/posthog/pull/12059). 
+
+## Takeaways
+
+One outcome of the offsite was a new focus on [nailing data exploration](handbook/people/team-structure/product-analytics#objective-1-nail-data-exploration). The team felt they had good data visualization tools, but not enough for data exploration. For example, it is difficult to move quickly between different types of data (events, persons, groups), often requiring recreating the same filters for multiple queries.
+
+Many of the projects related to this objective (on top of being fun and sparking joy).
+
+![Rome offsite team](../images/blog/rome-hackathon/rome-hackathon.jpeg)
+
+We like to be transparent with what we build. The details are found in the linked PRs as well as more expansively on the teams page in [our handbook](/handbook/people/team-structure/team-structure). We do this because we value feedback a lot. For example, users have been asking for better data exploration tools which helped it become a priority. 
```

Comment:
```suggestion
We like to be transparent with what we build. The details are found in the linked PRs as well as more expansively on the teams page in [our handbook](/handbook/people/team-structure/team-structure). We do this because we value feedback a lot. For example, users have been asking for better data exploration tools, which helped it become a priority. 
```